corehq/apps/domain/models.py

```diff
@@ -164,7 +166,16 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
     goal_followup_rate = DecimalProperty()
     # intentionally different from and commtrack_enabled so that FMs can change
     commtrack_domain = BooleanProperty()
-    performance_threshold = IntegerProperty()
+    performance_threshold = IntegerProperty(default=15)
+    experienced_threshold = IntegerProperty(default=3)
```

Comment:
One downside of defining these defaults is that if we ever later decided to change the thresholds we would have no way of determining the difference between "set to default" and "explicitly set to 15". if we think we will never change the defaults this doesn't matter and this change simplifies things.